tensorboard/webapp/metrics/types.ts

```diff
@@ -13,3 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 export * from './internal_types';
+
+// When adding a new value to the enum, please implement the deserializer on
```

Comment:
To be honest, I didn't realize that types is just reexporting internal_types. That's odd.

Maybe (as a followup PR, please not in this one) we should just get rid of internal_types and move everything into this file. The indirection is unnecessary and we shouldn't fool ourselves into thinking anything in internal_types is actually internal.

Reply:
+1 that totally makes sense as a followup